packages/core/src/GlobalStyles/GlobalStyles.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Global, css } from '@emotion/core';
+import emotionNormalize from 'emotion-normalize';
+import tokens from '@contentful/f36-tokens';
+
+export const GlobalStyles = ({
+  withNormalize = true,
+}: {
+  withNormalize?: boolean;
+}) => {
+  return (
+    <Global
+      styles={css`
+        ${withNormalize ? emotionNormalize : undefined};
+
+        html {
+          margin: 0;
+          margin: 0;
+          padding: 0;
+          border: 0;
+          min-height: 100%;
+        }
+
+        body {
+          font-size: ${tokens.fontSizeM};
+          font-family: ${tokens.fontStackPrimary};
+          color: ${tokens.gray800};
+          line-height: ${tokens.lineHeightDefault};
+          vertical-align: baseline;
```

Comment:
should we set: `boxSizing: "border-box"`?
It would fix the issue on the search bar:
![image](https://user-images.githubusercontent.com/1071799/132212970-ee47b28b-e71e-4926-9f6d-0bb55da5555b.png)

Reply:
Compose has the following code:

```
*,
html {
  box-sizing: border-box;
}
```

Since I'm CSS Junior Developer, I'd like to ask for opinions :)

Reply:
I think this would cover all elements and pseudo-elements and can be easily overwritten if needed:
```css
html {
  box-sizing: border-box;
}
*, *:before, *:after {
  box-sizing: inherit;
}
```

Reply:
+1 to ☝️